### ISSUE TITLE
Populate charts with real data.

### ISSUE THREAD

**Issue by devonoel** [GramBam/wealthtable#22]
Now that we have some pie charts setup, we'll want to populate with real data. That real data is going to come form the `UserDocuments`. We'll have to pull all the debts and assets from the `UserDocuments`, and build our pie charts form them.

**Comment by devonoel** [comment]
As an example, here's a breakdown of what I think will be involved for the "Assets" piechart you have.

First, you'll have to find all the `UserDocuments` for that user that are "Assets". An asset is anything with a positive `value`. So, when you're setting up the data in the `show` action of the controller, it'll probably look something like this:

```ruby
def show
  # Get all the user documents that have a value higher than 0
  asset_documents = current_user.user_documents.where("value > ?", 0)

  # Create a hash with all the names and values of each document
  @assets = {}
  asset_documents.each do |d|
    @assets[d.document.name] = d.value
  end
end
```

At the end, you'll end up with a variable `@assets` that has all the data you need for your first piechart, so you can just plug it into your first piechart.

The second one will be a little different, you'll have to get a total debt and total assets, and put those into your hash.

**Comment by devonoel** [comment]
Fixed in #69
